File: legacy/src/main/java/br/com/legacy/managers/dtos/UserCardDetails.java
package br.com.legacy.managers.dtos;

/**
 * Created by appsimples on 8/9/17.
 */

public class UserCardDetails {
    public String id;
    public String followerName;
    public String followerAvatar;
    public String userName;
    public String userAvatar;
    public Boolean followedByMe;

    // get the User Card EventSessionDetail of a friends list (different attribute names)

    public String userId;
    public String name;
    public String avatar;
    public boolean invitedByMe;

    // get the User Card EventSessionDetail from a club

    public String profilePicture;
    public String chatId;

}

File: component_user_session/src/main/kotlin/br/com/goin/component/session/user/UserCardDetails.kt
package br.com.goin.component.session.user

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class UserCardDetails(@SerializedName("id")
                           var id: String? = null,
                           @SerializedName("followerName")
                           var followerName: String? = null,
                           @SerializedName("followerAvatar")
                           var followerAvatar: String? = null,
                           @SerializedName("userName")
                           var userName: String? = null,
                           @SerializedName("userAvatar")
                           var userAvatar: String? = null,
                           @SerializedName("followedByMe")
                           var followedByMe: Boolean? = null,
                           @SerializedName("userId")
                           var userId: String? = null,
                           @SerializedName("name")
                           var name: String? = null,
                           @SerializedName("avatar")
                           var avatar: String? = null,
                           @SerializedName("invitedByMe")
                           var invitedByMe: Boolean = false,
                           @SerializedName("profilePicture")
                           var profilePicture: String? = null,
                           @SerializedName("chatId")
                           var chatId: String? = null) : Serializable